lib/extras/enc/apng.cc

```diff
@@ -169,8 +169,11 @@ Status EncodeImageAPNG(const CodecInOut* io, const ColorEncoding& c_desired,
     JXL_RETURN_IF_ERROR(ConvertToExternal(
         *transformed, bits_per_sample, /*float_out=*/false,
         c_desired.Channels() + (ib.HasAlpha() ? 1 : 0), JXL_BIG_ENDIAN, stride,
-        pool, raw_bytes.data(), raw_bytes.size(), /*out_callback=*/nullptr,
-        /*out_opaque=*/nullptr, metadata.GetOrientation()));
+        pool, raw_bytes.data(), raw_bytes.size(),
+        /*out_init_callback=*/nullptr,
```

Comment:
Should we make those into a struct perhaps?

Reply:
Sure. Only on the C++ side, right?

Reply:
Not in the API, if that's what you're asking :)